src/components/layouts/landingPage/ArticuleSection/index.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import HeadingText from 'Atoms/HeadingText';
+import Image from 'Atoms/Image';
+import __fontSizes from 'FontSizes';
+import __devices from 'Devices';
+
+const ArticulesSection = () => {
+  const ROBOT_IMAGE =
+    'https://res.cloudinary.com/maintenance-site/image/upload/v1580117293/Ojumo/2EJCSULRwC8.png';
+  const CHILDREN_IMAGE =
+    'https://res.cloudinary.com/maintenance-site/image/upload/v1580117292/Ojumo/AEaTUnvneik.png';
+  const articules = [
+    {
+      imageURL: ROBOT_IMAGE,
+      title: 'Effects of AI and ML',
+      body:
+        'Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum, modi quod dolorem cumque iure totam blanditiis accusantium amet quia facilis ipsum distinctio tempora? Soluta.',
+      date: 'Date - 20th May 2019',
+      time: 'Time - 8am - 4pm'
+    },
+    {
+      imageURL: CHILDREN_IMAGE,
+      title: 'Extracts from our visit to the ophanage',
+      body:
+        'Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum, modi quod dolorem cumque iure totam blanditiis accusantium amet quia facilis ipsum distinctio tempora? Soluta.',
+      date: 'Date - 20th May 2019',
+      time: 'Time - 8am - 4pm'
+    }
+  ];
+  return (
+    <ArticulesSection.Wrapper>
+      <HeadingText responsiveAlign='center'>Articules</HeadingText>
+      <ArticulesSection.Articule>
+        {articules.map(articule => (
+          <div key={articules.title} className='card'>
+            <div className='image'>
+              <Image
+                backgroundURL={articule.imageURL}
+                imgWidth='100%'
+                imgHeight='100%'
+              ></Image>
+            </div>
+            <div>
+              <h1>{articule.title}</h1>
+              <p className='date'>{articule.date}</p>
+              <p className='description'>{articule.body}</p>
+              <p className='read-more'> Read more...</p>
+            </div>
+          </div>
+        ))}
+      </ArticulesSection.Articule>
+    </ArticulesSection.Wrapper>
+  );
+};
+
+ArticulesSection.Articule = styled.div`
+  margin-top: 4%;
+  display: flex;
+
+  .content {
+    padding: 4% 0 7%;
+  }
+  .card {
+    display: flex;
+    margin-bottom: 4%;
+    flex-direction: column;
+    width: 45%;
+
+    .image {
+      width: 50vh;
+      height: 50vh;
+    }
+  }
+
+  h1 {
+    font-family: Montserrat;
+    font-style: normal;
+    font-weight: 800;
+    font-size: ${__fontSizes.point1_5};
+    line-height: 24px;
+    letter-spacing: 0.1em;
+    text-transform: uppercase;
+    margin: 20px 0;
```

Comment:
see fixed measurement feedback

Reply:
I don't understand this feedback?
Should I use the font-size fixed measurement for the margin and line spacing or should I just convert pixels to rem?

I currently converted to rem